### ISSUE TITLE
Feature Request: add some kind of zoning within AI Tool

### ISSUE THREAD

**Issue by eatoff8** [VorlonCD/bi-aidetection#34]
Use case: I want detections for people walking past, but I would also want an additional event (via mqtt or some such) when that person walks into my driveway.

**Comment by willngogit** [comment]
that seems like a pretty cool feature.  in the mean time, you can add the same camera a 2nd time with different masking to block out everything except your driveway.

**Comment by kalyway101** [comment]
Just a side note, but BI can technically already do this. You can set it up so MQTT messages are sent depending on the zone(s) that were triggered. 

So, for example, if a car sets off motion in the road (Zone A) then it won't send the request, but if motion is triggered in your driveway (Zone B) then it will send the message. Or only send it if the motion event was triggered in Zone A and Zone B. 

Though this does mean, in whatever platform your are using to check MQTT messages, you'll need to check motion zone AND if it was a relevant alert. Just depends on your use case though.

**Comment by SHerms** [comment]
I came across another variation of GentlePumpkin's original program. Looks like this one has that feature. Maybe some of these features can get merged in to VorlonCD's fork. I'm not sure how different they are as I haven't tried out @Ken98045 version.

https://github.com/Ken98045/On-Guard

https://ipcamtalk.com/threads/yet-another-free-extension-for-blue-iris-adding-ai-object-dectection-reduction-in-false-alarms-enhanced-notification-of-activity-on-guard.50660/

**Comment by SHerms** [comment]
Looks like the doc in the repo includes some screen shots. Program looks totally different. 
https://github.com/Ken98045/On-Guard/blob/master/OnGuard-ReadMe.docx

**Comment by DM-AU** [comment]
I would also appreciate a zoning feature. Blue Iris does it obviously but without AI it sets off too many false positives.

**Comment by Ken98045** [comment]
Have you tried my project On Guard at: https://github.com/Ken98045/On-Guard/releases/tag/1.4.6

 

Exactly what kind of zoning are you interested in?  On Guard supports a lot of zoning  (area of interest) options.  The project is largely stable, but there are new features being added at least every week, so that occasionally leads to problems.

**Comment by DM-AU** [comment]
Hi Ken,

I have not tried it yet, I am reading over the documentation etc as we speak. I was kind of hoping to not need to switch to another AI Tool just yet though as its all nicely configured. Would be good to just have it as a feature of this tool, but may need to switch if needed.

My goal is to have certain lights be triggered when AI motion is picked up in a certain area of the screen. This is only applicable to maybe 2 cameras but there will be about 5 zones that need AI detection.

**Comment by Ken98045** [comment]
Yes I support that using MQTT or ITTT or anything that will support url/http.  You can run off as well as on
